apps/src/templates/rubrics/teacherRubricRedux.ts

```diff
@@ -84,12 +127,98 @@ export const loadAllTeacherEvaluationData = createAsyncThunk(
   }
 );
 
+export const loadAiEvalStatusForAll = createAsyncThunk(
+  'teacherRubric/loadAiEvalStatusForAll',
+  async (params: {rubricId: number; sectionId: number}, thunkAPI) => {
+    const {rubricId, sectionId} = params;
+
+    const fetchAiEvaluationStatusAll = (
+      rubricId: number,
+      sectionId: number
+    ) => {
+      return fetch(
+        `/rubrics/${rubricId}/ai_evaluation_status_for_all?section_id=${sectionId}`
+      );
+    };
+
+    fetchAiEvaluationStatusAll(rubricId, sectionId).then(response => {
+      if (response.ok) {
+        response.json().then(data => {
+          thunkAPI.dispatch(setAiEvalStatusMap(data.aiEvalStatusMap));
+          delete data.aiEvalStatusMap;
+          thunkAPI.dispatch(setAiEvalStatusCounters(data));
+        });
+      }
+    });
+  }
+);
+
+const computeLevelStatus = (level: LevelWithProgress | undefined) => {
```

Comment:
If this function and `computeBubbleStatus` have no relation to the redux store, what do you think about moving them to a utils file instead?